cmd/cli/app/profile/list.go

```diff
@@ -90,6 +90,7 @@ func listCommand(ctx context.Context, cmd *cobra.Command, _ []string, conn *grpc
 func init() {
 	ProfileCmd.AddCommand(listCmd)
 	listCmd.Flags().StringP("label", "l", "", "Profile label to filter on")
+	listCmd.Flags().MarkHidden("label")
```

Comment:
The linter is flagging an unhandled error here. Can you receive an error and exit if error isn't nil?

Reply:
Yep, have just pushed up 👍